packages/ui-library/src/components/loader/index.ts

```diff
@@ -1,36 +1,38 @@
 import { html } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { property } from 'lit/decorators.js';
-import { styleCustomLight, styleCustomDark } from './index.css';
+import { staticStyles } from './index.css';
 
 import { TAG_NAME } from './renderFunction';
 import { ThemeType } from '../../foundation/_tokens-generated/index.themes';
 import { ActionSizesType, FeedbackVariantType } from '../../globals/types';
 import { LitElementCustom } from '../../utils/lit-element-custom';
 
 export class BlrLoader extends LitElementCustom {
-  static styles = [];
+  static styles = [staticStyles];
 
   @property() sizeVariant?: ActionSizesType = 'md';
   @property() variant?: FeedbackVariantType;
   @property() theme: ThemeType = 'Light';
 
   protected render() {
     if (this.sizeVariant) {
-      const dynamicStyles = this.theme === 'Light' ? [styleCustomLight] : [styleCustomDark];
+      const containerClasses = classMap({
+        'loader-container': true,
+        [`${this.sizeVariant}`]: this.sizeVariant,
```

Comment:
Please make this consistent with other i.e. without template literals. Same for line 23